### ISSUE TITLE
Implement Documentation for Adapter Lifecycle Support

### ISSUE THREAD

**Issue by tp1de** [ioBroker/dev-docs#39]
I recognized quite some difficulties to collect all the information needed for updating adapters during the Life-Cycle and new releases in respect to ioBroker base system. Especially for those developers who are not IT guys and/or doing changes only every couple of months.

I believe that it is important to secure mid- and long-term support for adapters with minimal efforts.
As far as I can seen adapter adjustments might be needed due to:

1. new nodejs versions
2. new dependencies / version upgrades
3. adjustments to (release) scripts
4. code-adjustments due to new js-controller versions
5. code adjustments due to other adapter changes (sql / history ..)
6. etc...

Actually it is quite difficult for non-insiders to collect the necessary information and recognize the need for change.
I would recommend to have a documentation prepared within the ioBroker Developer Portal. Additionally there should an alert system towards the owners of adapters established about necessary adjustments and how to do it.

**Comment by Garfonso** [comment]
Why did you close this issue?

**Comment by tp1de** [comment]
One year without any reaction...

**Comment by Apollon77** [comment]
Yes because it seems that since one year noone had time. The idea still makes sense so we shoukld keep it open. Maybe times change and we get resources again available
